test-cases/longevity/longevity-parallel-schema-changes-12h.yaml

```diff
@@ -1,21 +1,20 @@
-             "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=50 -clustering-row-count=2500 -partition-offset=51  -clustering-row-size=uniform:1024..4096  -concurrency=10  -connection-count=10  -consistency-level=quorum -rows-per-request=10 -timeout=30s -retry-number=30 -retry-interval=80ms,1s -iterations 0 -duration=700m",
-             "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=50 -clustering-row-count=2500 -partition-offset=151 -clustering-row-size=uniform:4096..8192  -concurrency=10  -connection-count=10  -consistency-level=quorum -rows-per-request=10 -timeout=30s -retry-number=30 -retry-interval=80ms,1s -iterations 0 -duration=700m",
-             "scylla-bench -workload=sequential -mode=read  -replication-factor=3 -partition-count=50 -clustering-row-count=5000 -partition-offset=11  -clustering-row-size=uniform:1024..4096   -concurrency=100 -connection-count=100 -consistency-level=quorum -rows-per-request=10 -timeout=30s -retry-number=30 -retry-interval=80ms,1s -iterations 0 -duration=700m -validate-data",
-             "scylla-bench -workload=sequential -mode=read  -replication-factor=3 -partition-count=50 -clustering-row-count=5000 -partition-offset=121 -clustering-row-size=uniform:4096..8192   -concurrency=100 -connection-count=100 -consistency-level=quorum -rows-per-request=10 -timeout=30s -retry-number=30 -retry-interval=80ms,1s -iterations 0 -duration=700m -validate-data"
+stress_cmd: [
+             "cassandra-stress write cl=QUORUM duration=720m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate 'threads=40 throttle=15000/s' -pop 'dist=uniform(1..20971520)' -col 'n=FIXED(10) size=FIXED(128)' -log interval=10 -errors 'retries=30'",
+             "cassandra-stress read  cl=QUORUM duration=720m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate 'threads=40 throttle=15000/s' -pop 'dist=uniform(1..20971520)' -col 'n=FIXED(10) size=FIXED(128)' -log interval=10 -errors 'retries=30'",
+             "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=50 -clustering-row-count=1000 -partition-offset=11 -clustering-row-size=uniform:1024..4096  -concurrency=20  -connection-count=20  -consistency-level=quorum -rows-per-request=50 -timeout=60s -retry-number=30 -retry-interval=500ms,1s -iterations 0 -duration=700m -max-rate 1000",
+             "scylla-bench -workload=sequential -mode=read  -replication-factor=3 -partition-count=50 -clustering-row-count=1000 -partition-offset=41 -clustering-row-size=uniform:1024..4096   -concurrency=20 -connection-count=20  -consistency-level=quorum -rows-per-request=50 -timeout=60s -retry-number=30 -retry-interval=500ms,1s -iterations 0 -duration=700m -max-rate 1000"
 
              ]
 
 n_db_nodes: 5
-n_loaders: 3
+n_loaders: 2
 n_monitor_nodes: 1
 
 instance_type_db: 'i3en.xlarge'
```

Comment:
@aleksbykov let's change it to i4i.something.

Unless there is a demand for large dataset here.
The i4i will better handle the load.

Reply:
@roydahan  changed.